src/main/java/sirius/biz/scripting/mongo/MongoCustomScriptController.java

```diff
@@ -76,6 +76,9 @@ public void editScript(WebContext webContext, String id) {
     @Routed("/scripting/scripts/:1/delete")
     @Permission(ScriptingController.PERMISSION_SCRIPTING)
     public void deleteScript(WebContext webContext, String id) {
+        if (!webContext.isSafePOST()) {
+            return;
+        }
         deleteEntity(webContext, tryFindForTenant(MongoCustomScript.class, id));
```

Comment:
Would invert the if and wrap it around the `deleteEntity` so the redirect does happen and the browser won't wait for a response